test/e2e/csv_e2e_test.go

```diff
@@ -28,7 +27,7 @@ var immediateDeleteGracePeriod int64 = 0
 
 func buildCSVCleanupFunc(t *testing.T, c operatorclient.ClientInterface, crc versioned.Interface, csv v1alpha1.ClusterServiceVersion, namespace string, deleteCRDs bool) cleanupFunc {
 	return func() {
-		require.NoError(t, crc.OperatorsV1alpha1().ClusterServiceVersions(namespace).Delete(csv.GetName(), &metav1.DeleteOptions{}))
+		crc.OperatorsV1alpha1().ClusterServiceVersions(namespace).Delete(csv.GetName(), &metav1.DeleteOptions{})
```

Comment:
if you do `assert.NoError` instead of `require`, it will keep executing the entire test and at the end it will report that the delete failed (I assume you removed it because it was stopping execution?)

Reply:
I removed the `require.NoError` call because it didn't make sense to me for a test failure to be caused by an issue deleting a CSV (especially since other deferred cleanup functions could possibly delete the CSV in question e.g. installplan cleanups).